pkgs/by-name/tp/tpi/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage {
+  pname = "tpi";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "turing-machines";
+    repo = "tpi";
+    rev = "v1.0.6";
```

Comment:
Usually this would be "v${version}" or similar

Reply:
I wasn't aware that I hadn't set my mail address. I use git cmd and set it for each repo separate because I have different mail addresses for GitHub, GitLab and Bitbucket as well as work and private. If I want to apply your suggestions is it preferable to create a new commit or squash the changes in the original commit?

Reply:
You can apply all suggestions and have a single commit `tpi: init at 1.0.6`

Reply:
Okay, I will do that and add my email address